charts/smb-listener/values.yaml

```diff
@@ -22,6 +22,7 @@ nats:
 # This sets the smb-listener container image
 dockerRegistry: "us-docker.pkg.dev/gcnv-ai-dev/genai-toolkit/"
 image:
+  tag: ""
```

Comment:
I like this whole concept of keeping the images in line with the chart version for releases. It's super lightweight to release new chart versions so why not just have a new one with every image release? This also makes tools like argo a lot happier as you'll never have the same "version" of a chart with deployments that reference different images.

Reply:
this tag is only if you want to override the version, the version tag by the image is by default the helm chart version